#Standard header
Header header

#######################
# Telemetry
#######################
# Linear speeds of the motors
float32 mot_speed_left
float32 mot_speed_right
# PWM values
int32 mot_pwm_left
int32 mot_pwm_right
# RPM values
float32 mot_rpm_left
float32 mot_rpm_right
# Battery charge
float32 battery
